bin/oay/src/services/webdav/webdav_file.rs

```diff
@@ -83,7 +83,7 @@ impl DavFile for WebdavFile {
     }
 }
 
-fn convert_error(opendal_error: opendal::Error) -> dav_server::fs::FsError {
+pub fn convert_error(opendal_error: opendal::Error) -> dav_server::fs::FsError {
```

Comment:
We can implement `From<opendal::Error> for dav_server::fs::FsError`

Reply:
Implementing traits requires compliance with the Orphan Principle, seem oay is not in the same crate as opendal?

Reply:
or make a newtype?

Reply:
Oh, you are right.